docs/pages/docs/build-platforms/github-actions.mdx

```diff
@@ -78,12 +90,14 @@ If you are having problems make sure you have done the following:
 ### Running With Branch Protection
 
 GitHub actions require a little more setup to use `auto` with branch protection.
+We can't use the `GITHUB_TOKEN` to commit to the branch because it does not have high enough permissions.
+You will need to create a token with admin access to your repo and modify your build config like the following:
```

Comment:
Might be worth expanding "admin access" to the specific scopes. At a minimum, you need the full `repo` scope, plus the `packages` (read + write) scope on the token.